python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -130,9 +131,14 @@ def runtimeEvalString(this_obj, attr_name, val):
 
     return new_val
 
-
+_stored_evals = {}
 
 def raw_eval(val):
+    if val not in _stored_evals:
+        _stored_evals[val] = _really_eval(val)
+    return copy.deepcopy(_stored_evals[val])
+
+def _really_eval(val):
```

Comment:
Can we avoid adding more function names like `_really_do_something`. The split is arbitrary and it makes tracking the code much more difficult.

Also, any new functions must have full docstrings.